tests/src/test/scala/whisk/core/database/test/ReplicatorTests.scala

```diff
@@ -130,6 +130,17 @@ class ReplicatorTests extends FlatSpec
         }
     }
 
+    // Do cleanups of possibly existing databases
+    val dbs = replicatorClient.dbs().futureValue
+    dbs shouldBe 'right
+    dbs.right.get.filter(dbname => dbname.contains(testDbPrefix)).map(dbName => removeDatabase(dbName))
+
+    val docs = replicatorClient.getAllDocs().futureValue
+    docs shouldBe 'right
+    docs.right.get.fields("rows").convertTo[List[JsObject]]
+        .filter(_.fields("id").convertTo[String].contains(testDbPrefix))
```

Comment:
Should that be `startsWith(testDbPrefix)` for good measure? Might need to do `startsWith("backup_" + testDbProfix) || startsWith("continuous_" + testDbPrefix)`